final_project/graph/graph.py

```diff
@@ -151,7 +152,7 @@ def connected_components(graph: nx.Graph) -> nx.Graph:
     else:
         connected_components = nx.connected_components
 
-    largest_component = sorted(connected_components(graph), key=len, reverse=True)[0]
+    largest_component = sorted(nx.weakly_connected_components(graph), key=len, reverse=True)[0]
```

Comment:
`connected_components` jako aliast na jedną z funkcji działało? Teraz funkcja będzie działać tylko dla `DiGraph`, a tak była też dla undirected.

Reply:
a to sorki, bo też zmieniłam to na chwilę żeby sprawdzić czy jest różnica jak to działa i zapomniałam tu zmienić z powrotem! Oczywiście może być tak jak było wcześniej, sorki za problem